test_car.py

```diff
@@ -250,10 +250,10 @@ def test_braking_deceleration_forward(car_object_basic):
     car_object_basic.accelerate_forward(2.5,5)
     car_object_basic.braking_deceleration(1, 3)
     assert car_object_basic.check_velocity() == 9.5
-    assert car_object_basic.check_distance() == 565.75
+    assert car_object_basic.check_distance() == 573.25
     car_object_basic.braking_deceleration(200, 2)
     assert car_object_basic.check_velocity() == 0
-    assert car_object_basic.check_distance() == 591.875
+    assert car_object_basic.check_distance() == 601.75
```

Comment:
I get this to be 573.25 (maybe something got switched up since this matched the previous one)